source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/list/index.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { compose, withProps } from 'recompose';
+
+import { query, buildOptions, withLoader, WorkflowDefinitionResource, attributesFor } from '@data';
+import { withTranslations, i18nProps } from '@lib/i18n';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+
+import { addWorkflowDefinitionPathName } from '../index';
+import WorkflowDefinitionItem from './item';
+import { compareVia } from '@lib/collection';
+
+interface IOwnProps {
+  workflowDefinitions: WorkflowDefinitionResource[];
+}
+
+type IProps =
+  & IOwnProps
+  & i18nProps
+  & RouteComponentProps<{}>;
+
+class ListWorkflowDefinition extends React.Component<IProps> {
+
+  showAddForm = (e) => {
+    e.preventDefault();
+    const { history } = this.props;
+    history.push(addWorkflowDefinitionPathName);
+  }
+
+  render() {
+
+    const { t, workflowDefinitions } = this.props;
+
+    return (
+      <>
+        <h2 className='sub-page-heading'>
+          {t('admin.settings.workflowDefinitions.title')}
+        </h2>
+        <div className='m-b-xxl'>
+          <button
+            className='ui button tertiary uppercase large m-b-lg'
+            onClick={this.showAddForm}
+          >
+            Add Workflow Definition
```

Comment:
translation missing

Reply:
I noticed this one when I first entered it and then couldn't find it anymore.  Thought I must have already fixed it.